Booting/linux-bootstrap-1.md

```diff
@@ -97,7 +101,7 @@ SECTIONS {
 }
 ```
 
-Now BIOS has started to work. After all initializations and hardware checking, it needs to load operating system. BIOS tries to find bootable device which contains boot sector. Boot sector is the first sector on device (512 bytes) and contains sequence of `0x55` and `0xaa` at 511 and 512 byte. For example:
+Now BIOS has started to work. After all initializations and hardware checking, it needs to load the operating system. BIOS tries to find a bootable device which contains a boot sector. A boot sector is the first sector on the device (512 bytes) and contains a sequence of `0x55` and `0xaa` at 511 and 512 byte. For example:
```

Comment:
BIOS isn't a proper noun, it's an acronym for a common noun (there are numerous BIOSes). Therefore, it should be referred to as 'the BIOS'.